### PR TITLE
Fix terrain curve plotting

### PR DESCRIPTION
**Pull request by Vagabond**
Try to make the terrain curve plotting more sane, still has some weird behaviour sometimes.

### CODE REVIEW

File: src/outputs.cc [hunk 1]
```diff
@@ -1230,13 +1230,16 @@ void SeriesData(struct site source, struct site destination, unsigned char fresn
...
+    struct site mp;
 
     bool has_clutter = false, has_fresnel = false;
 
     ReadPath(destination, source, out);
     azimuth = Azimuth(destination, source);
     distance = Distance(destination, source);
     refangle = ElevationAngle(destination, source);
+    printf("refangle is %f\n", refangle);
+    refangle = 0.0;
```
Comment (JayKickliter) [inline src/outputs.cc:1242]: why zeroize immediately?

Comment (Vagabond) [reply src/outputs.cc:1242]: this was all experimental, I was *trying* to center the peak of the earth curve, but zeroizing at least let me get the transmitter (called the destination here, for some reason) at the top of the curve, most of the time.

File: src/outputs.cc [hunk 2]
```diff
@@ -1264,15 +1269,39 @@ void SeriesData(struct site source, struct site destination, unsigned char fresn
...
+    }
+
     for (x = 0; x < out->path.length - 1; x++) {
         remote.lat = out->path.lat[x];
         remote.lon = out->path.lon[x];
         remote.alt = 0.0;
         terrain = GetElevation(remote);
-        if (x == 0) terrain += destination.alt; /* RX antenna spike */
+        if (x == 0) terrain += destination.alt;               /* RX antenna spike */
+        if (x == out->path.length - 1) terrain += source.alt; /* TX antenna spike */
```
Comment (JayKickliter) [inline src/outputs.cc:1292]:
```suggestion
```

File: src/outputs.cc [hunk 3]
```diff
@@ -1298,6 +1326,8 @@ void SeriesData(struct site source, struct site destination, unsigned char fresn
             if ((lr.frq_mhz >= 20.0) && (lr.frq_mhz <= 100000.0) && fresnel_plot) {
                 f_zone += r;
                 fpt6_zone += r;
+                f_zone += (height - terrain);
```
Comment (Vagabond) [inline src/outputs.cc:1329]: This may be wrong

**Comment by JayKickliter** [comment]
Closing in favor of #40

**Pull request closed without merging**